test/framework/src/main/java/org/elasticsearch/test/rest/RestTestLegacyFeatures.java

```diff
@@ -59,6 +59,8 @@ public class RestTestLegacyFeatures implements FeatureSpecification {
     public static final NodeFeature ML_INDICES_HIDDEN = new NodeFeature("ml.indices_hidden");
     @UpdateForV9
     public static final NodeFeature ML_ANALYTICS_MAPPINGS = new NodeFeature("ml.analytics_mappings");
+    @UpdateForV9
```

Comment:
How does this need to be updated?

Reply:
It will be removed: the cluster restart tests will be from 8.0.0 onwards, once we have 9.x. 
Now that you ask I'm second guessing this: will it be true?

Reply:
That's fine, it just needs a comment on it

Reply:
Ah ok, got it! I will add the comment (to the others as well)

Reply:
Since most of these features will need to be removed after we move to v9, I have moved the annotation to the class, with an appropriate comment. However, I think it's better if this goes into a separate PR so this one focuses on the fix. What do you think?

Reply:
Sounds good to me